### PR TITLE
Update to latest metrics-server 0.3.x works

### PR DESCRIPTION
**Pull request by krsna1729**
With kubelet service file fixed and correct args set in
metrics-server-deployment.yaml, autoscale seems to be working reliably.

Fixes: #5

Signed-off-by: Saikrishna Edupuganti <saikrishna.edupuganti@intel.com>

### CODE REVIEW

**Comment by krsna1729** [comment]
```
$ cat /etc/os-release
NAME="Clear Linux OS"
VERSION=1
ID=clear-linux-os
ID_LIKE=clear-linux-os
VERSION_ID=27700
PRETTY_NAME="Clear Linux OS"
```

**Comment by krsna1729** [comment]
```
./create_stack.sh minimal
./tests/autoscale/test-autoscale.sh
```

File: clr-k8s-examples/tests/autoscale/load-gen.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
```
Comment (mcastelino) [inline clr-k8s-examples/tests/autoscale/load-gen.yaml]: @krsna1729 should we make this a deployment. That way we can scale it up to generate more load?

File: clr-k8s-examples/1-core-metrics/metrics-server-deployment.yaml [hunk 2]
```diff
@@ -22,10 +23,19 @@ spec:
...
       - name: metrics-server
-        image: gcr.io/google_containers/metrics-server-amd64:v0.2.0
+        image: k8s.gcr.io/metrics-server-amd64:v0.3.1
         imagePullPolicy: Always
-        command:
-        - /metrics-server
-        - --source=kubernetes.summary_api:https://kubernetes.default.svc?kubeletHttps=true&kubeletPort=10250&useServiceAccount=true&insecure=true
+        args:
+        - --logtostderr
+        - --kubelet-insecure-tls
```
Comment (mcastelino) [inline clr-k8s-examples/1-core-metrics/metrics-server-deployment.yaml]: @krsna1729 is this a limitation. This was due to a bug right.

**Comment by mcastelino** [comment]
@ganeshmaharaj @krsna1729 can we spin a new release of this package on Clearlinux. This gives us the the metrics server we wanted. We had downgraded the metrics server due to issues we had seen.

File: clr-k8s-examples/1-core-metrics/metrics-server-deployment.yaml (hunk 2 above)
Comment (krsna1729) [reply clr-k8s-examples/1-core-metrics/metrics-server-deployment.yaml]: This was because of self-signed certs IIRC

**Pull request merged**